tests/integration_tests/test_source_selection.py

```
@@ -0,0 +1,330 @@
+
+    @pytest.fixture
+    def collection(self, client):
+        """Create test collection with sample data"""
+        collection_name = "test_source_selection"
+
+        # Clean up if exists
+        try:
+            if client.has_collection(collection_name):
+                client.delete_collection(collection_name)
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)